components/application-connectivity-certs-setup-job/README.md

```diff
@@ -0,0 +1,16 @@
+# Application Connectivity Certs Setup Job
+
+## Overview
+
+Application Connectivity Certs Setup Job populates Kubernetes Secrets with the provided certificate and key pair or generates new one.
+
+
+## Configuration
+
+The Application Gateway has the following parameters:
+**connectorCertificateSecret** - "Secret namespace/name where the Connector Service client certificate and key are kept.
+**caCertificateSecret** - "Secret namespace/name where CA certificate is kept.
```

Comment:
If you pass both the CA cert name and Namespace, please describe the format that must be passed. Capitalize "Namespace". Change "kept" to "stored".

Reply:
Done